Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+- Backup and restore
+  - It is important for disaster recovery and data migration scenarios
+- Monitoring
+  - Observability is a key feature for operating a service in production so the container registry should expose key metrics
+- HA mode
+  - Ensure system uptime even in the event of a failure
+- Registry replication
+  - Replication allows users to replicate container images between registries of the same instances and between registries of different instances as well
+- Proxy cache (pull-through cache)
+  - Proxy cache allows you to use a container registry to proxy and cache images from a target public or private registry
+- Quota management
+  - Control over resource use
+- Garbage collection
+  - Removing blobs from the filesystem when they are no longer referenced by a manifest
+- Retention policy
+  - Reduce the number of image tags, many of which might not be required after a given time or once a subsequent image tag has superseded them
+
+**Desirable features**
+
+- Additionally supported artifacts
+  - Additional artifacts that the registry is able to store in addition to OCI artifacts, e.g. Java, Node.js, or Python packages
+- Integration possibilities
+  - Ability to cooperate with another software solution in order to improve own feature set (e.g. integration of P2P solution for improving container image distribution (download speed and stability, high scalability ...))
+
+Refer to the table of evaluated projects with their features. Note that only container
+registry implementations that passed the OSS health stage (Harbor, Quay, and Dragonfly)
+are evaluated here.
+
+| Features                            | Harbor                                        | Quay                                                                | Dragonfly                    |
+|-------------------------------------|-----------------------------------------------|---------------------------------------------------------------------|------------------------------|
+| Audit Logs                          | ✓                                             | ✓                                                                   | ✗                            |
+| Authentication of system identities | ✓ Robot Accounts                              | ✓ Robot Accounts                                                    | ✗                            |
+| Authentication of users             | ✓ Local database, LDAP, OIDC, UAA             | ✓ Local database, LDAP, Keystone, JWT                               | ✓ Local database             |
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
```

Comment:
Harbor supports Webhooks following [CloudEvents](https://cloudevents.io/) spec (next release)

Reply:
I incorporated it in 02382f14b5ba7204845eb3e690c4d5b78a49770b